src/test/java/net/datafaker/base/PhoneNumberTest.java

```diff
@@ -121,11 +125,98 @@ void testPhone_CA() {
     @RepeatedTest(10)
     void testAllCellPhone_enPh() throws NumberParseException {
         final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
-        String phoneNumber = new BaseFaker(new Locale("en_PH")).phoneNumber().phoneNumber();
+        String phoneNumber = new BaseFaker(new Locale("en", "PH")).phoneNumber().phoneNumber();
         Phonenumber.PhoneNumber proto = util.parse(phoneNumber, "PH");
         assertThat(util.isValidNumberForRegion(proto, "PH")).as(phoneNumber).isTrue();
     }
 
+    @ParameterizedTest
+    @MethodSource("generateLanguageAndRegionOfLocales")
+    void testAllPhoneNumberNational(Locale locale, String phoneNumberRegion) throws NumberParseException {
+        final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
+        int errorCount = 0;
+        final BaseFaker faker = new BaseFaker(locale);
+        for (int i = 0; i < 10; i++) {
+            String phoneNumber = faker.phoneNumber().phoneNumber();
+            Phonenumber.PhoneNumber proto = util.parse(phoneNumber, phoneNumberRegion);
+            if (!util.isValidNumberForRegion(proto, phoneNumberRegion)) {
+                errorCount++;
+            }
+        }
+        assertThat(errorCount).isLessThan(250);
+    }
+    @ParameterizedTest
+    @MethodSource("generateLanguageAndRegionOfLocales")
+    void testAllPhoneNumberInternational(Locale locale, String phoneNumberRegion) throws NumberParseException {
+        final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
+        int errorCount = 0;
+        final BaseFaker faker = new BaseFaker(locale);
+        for (int i = 0; i < 10; i++) {
+            String phoneNumber = faker.phoneNumber().phoneNumberInternational();
+            Phonenumber.PhoneNumber proto = util.parse(phoneNumber, phoneNumberRegion);
+            if (!util.isValidNumberForRegion(proto, phoneNumberRegion)) {
+                errorCount++;
+            }
+        }
+        assertThat(errorCount).isLessThan(250);
+    }
```

Comment:
Could somebody check if it's correct? Because if to add to `private static Stream<Arguments> generateLanguageAndRegionOfLocales()` `Arguments.of(new Locale("fr", "CH"), "CH")` it runs correctly, but it cannot run correctly, because there's no `formats_international` in `fr-CH.yml`